src/Xamarin.Android.Build.Tasks/Tasks/BuildApk.cs

```diff
@@ -380,34 +391,36 @@ static Regex FileGlobToRegEx (string fileGlob, RegexOptions options)
 			return new Regex (sb.ToString (), options);
 		}
 
-		void AddAssemblies (ZipArchiveEx apk, bool debug, bool compress, IDictionary<string, CompressedAssemblyInfo> compressedAssembliesInfo, string assemblyStoreApkName)
+		void AddRuntimeConfigBlob (ZipArchiveEx apk)
```

Comment:
Is this method always required? Not kust for AssemblyStore ?

e.g do we need this in Debug/Fast Deployment modes?

Reply:
We always put the runtime config blob (produced from the JSON input file) in the apk, yes. This is the MonoVM settings/feature flags/properties etc, unrelated to our assembly blobs.